core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -2197,6 +2197,33 @@ public void testLoadMetadata() throws Exception {
             .setLoadDescendantType(DescendantType.ONE));
   }
 
+  /**
+   * Tests ufs load with ACL.
+   * Currently, it respects the ufs permissions instead of the default ACL for loadMetadata.
+   * We may change that in the future, and change this test.
+   * TODO(david): update this test when we respect default ACL for loadmetadata
+   */
+  @Test
+  public void loadMetadataWithACL() throws Exception {
+    FileUtils.createDir(Paths.get(mUnderFS).resolve("a").toString());
+    AlluxioURI uri = new AlluxioURI("alluxio:/a");
```

Comment:
`alluxio:/a` doesn't look like a valid uri?

Reply:
it is technically valid and found at other locations in this file, but I removed the scheme `alluxio:` just to make it clear.